starter-core/src/main/java/io/micronaut/starter/feature/build/maven/templates/pom.rocker.raw

```diff
@@ -58,7 +58,11 @@ MavenBuild mavenBuild
     <micronaut.runtime>azure_function</micronaut.runtime>
 }
 @if (applicationType == ApplicationType.DEFAULT && features.contains("aws-lambda")) {
-    <micronaut.runtime>lambda</micronaut.runtime>
+  @if (features.contains("graalvm")) {
+    <micronaut.runtime>lambda_provided</micronaut.runtime>
+  } else {
+    <micronaut.runtime>lambda_java</micronaut.runtime>
+  }
```

Comment:
This will fail with Maven, since those values don't exist in this enum: https://github.com/micronaut-projects/micronaut-maven-plugin/blob/master/src/main/java/io/micronaut/build/MicronautRuntime.java

Reply:
This is blocked via: https://github.com/micronaut-projects/micronaut-maven-plugin/issues/321